ts/src/scenario.ts

```diff
@@ -140,11 +144,13 @@ export class Scenario {
   ): Promise<Player[]> {
     await this.ensureLocalServices();
     return Promise.all(
-      new Array(amount).fill(0).map(async () => {
-        const conductor = await this.addConductor();
-        if (networkConfig) {
-          conductor.setNetworkConfig(networkConfig);
-        }
+      new Array(amount).fill(0).map(async (_, i) => {
+        // Wait 1s between conductor launches.
+        // This is a workaround to avoid connection failures.
+        // See https://github.com/holochain/tryorama/issues/297
+        await pause(i * 1000);
+
+        const conductor = await this.addConductor(networkConfig);
```

Comment:
How much does this affect the networking behavior? I would expect Holochain not to do any networking until a space has been created, which only happens when a cell is created.

Reply:
Yes good catch -- removed from this function: https://github.com/holochain/tryorama/pull/303/commits/e51ba0baf0c795dd9a7d2836ae07ec888052e3a5